docs/installing-solidity.rst

```diff
@@ -6,6 +6,17 @@
 Installing Solidity
 ###################
 
+Versioning
+==========
+
+Solidity versions follow `semantic versioning <https://semver.org>` and in addition to
+releases, **nightly development builds** are also made available.  The nightly builds
+are not guaranteed to be working and despite best efforts they might contain undocumented
+and/or broken changes. We recommend to use the latest release. All the method below will
```

Comment:
Had to reword it now a bit because there's also the source installation guide and that wont give a release.